nom-sql/src/show.rs

```diff
@@ -19,7 +19,7 @@ pub enum ShowStatement {
     Events,
     Tables(Tables),
     CachedQueries(Option<QueryID>),
-    ProxiedQueries(Option<QueryID>),
+    ProxiedQueries(ProxiedQueriesOptions),
```

Comment:
I'm happy with this merging as-is, but it is worth noting that I'd have been happy with this being just a regular struct enum variant rather than a standalone struct:

```suggestion
    ProxiedQueries {
        pub query_id: Option<String>,
        pub only_supported: bool,
    }
```

Reply:
mostly since you said you're new to rust; wanted to let you know this was possible 😉

Reply:
Thanks @glittershark .